multipart/multipart.py

```diff
@@ -110,6 +110,8 @@ def parse_options_header(value: Union[str, bytes]) -> Tuple[bytes, Dict[bytes, b
     options = {}
     for param in params:
         key, value = param
+        if isinstance(value, tuple):
+            value = value[-1]
```

Comment:
Can you add a comment about this, please?

Reply:
Sure thing!